src/components/common/styles.js

```diff
@@ -99,7 +119,7 @@ export const StyledDot = styled.div`
   border-radius: 1rem;
 `;
 
-/* login page styled components */
+/* landing page styled components */
 export const StyledWrapper = styled.section`
```

Comment:
Can this get a more descriptive name?

Reply:
I changed it to StyledPopupContainer as it's the container for the login, sign up, and demo popups.

Reply:
I also changed the names of other styled components which I felt I didn't make descriptive enough.

Reply:
If youre using this for the login/signup popups, you could probably use StyledPopup, which is used for the add stack and edit stack menus.